src/ansible/roles/common/tasks/main.yml

```diff
@@ -65,9 +65,19 @@
     mode: '0600'
   with_items:
   - { src: 'root.id_rsa', dest: 'id_rsa' }
-  - { src: 'root.id_rsa.pub', dest: 'authorized_keys' }
+#  - { src: 'root.id_rsa.pub', dest: 'authorized_keys' }
   - { src: 'root.id_rsa.pub', dest: 'id_rsa.pub' }
 
+- name: Append or create authorized_keys
```

Comment:
Ok, but why?

Reply:
Because this code just overwrote the authorized keys ignoring that there are already authorized_keys from idmci. This way it works nicely together instead of breaking the idmci tooling.